src/olympia/amo/sitemap.py

```diff
@@ -92,6 +97,45 @@ def location(self, item):
         return reverse(item)
 
 
+class CategoriesSitemap(Sitemap):
+    priority = 0.7
+    # i18n = True  # TODO: support all localized urls
+    changefreq = 'always'
+    lastmod = datetime.datetime.now()
+
+    def items(self):
+        def additems(type):
+            items = []
+            for category in CATEGORIES[current_app.id][type].values():
+                items.append((category, 1))
+                pages_needed = math.ceil(addon_counts.get(category.id, 1) / page_size)
+                for page in range(2, pages_needed + 1):
+                    items.append((category, page))
+            return items
+
+        page_size = settings.REST_FRAMEWORK['PAGE_SIZE']
```

Comment:
I don't think that's the case for category pages but the frontend is free to use a different `page_size` value. I took a quick look and we probably don't set it very often in addons-frontend but that's not unlikely.

If we start to rely on this value here, we'll probably want tests in addons-frontend to avoid any changes around `page_size`.

Reply:
> I don't think that's the case for category pages but the frontend is free to use a different `page_size` value. I took a quick look and we probably don't set it very often in addons-frontend but that's not unlikely.

So, what change are you proposing exactly?  We obviously need to assume a page_size to calculate how many pages.

> If we start to rely on this value here, we'll probably want tests in addons-frontend to avoid any changes around `page_size`.

perhaps, but that equally applies to any url that frontend serves.  Is the ask here to log an issue to add a test to frontend?

Reply:
Writing tests to prevent regressions is what I am suggesting.

Reply:
mozilla/addons-frontend#10498